### PR TITLE
Feature/faas samples

### PR DESCRIPTION
**Pull request by dahby**
Updated folder with functions and readme's for two guides on Talkyard.

- [Caching Current Queue Health APIs w/ FaaS & the Conversation Context Service](https://talkyard.livepersonai.com/-18/caching-current-queue-health-apis-w-faas-the-context-session-store)
- [Caching Shift Status API Data w/ FaaS & Conversation Context Service](https://talkyard.livepersonai.com/-24/caching-shift-status-api-data-w-faas-conversation-context-service)

### CODE REVIEW

File: faas-samples/current-queue-health/README.md [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+
+This process was originally documented on our [developers forum](https://talkyard.livepersonai.com/-18/caching-current-queue-health-apis-w-faas-the-context-session-store).
+
+---
+
+LivePerson provides a number of APIs which are designed to be used with Reporting Dashboards to inform on items such as the status of the queue and the shift status of agents. Having this information readily accessible within Conversation Builder can assist with correctly setting expectations or properly routing individuals when a skill may not be available to escalate to. Unfortunately, many of these reporting dashboard APIs are not intended for use at a Conversational Scale, and doing so could be detrimental to the stability of our platform.
+
+This guide is intended to provide an alternative to calling these APIs at every conversation. By using several Conversational Cloud services in conjunction with one another to regularly cache needed data, we can make this information accessible in a more robust service that is properly scaled to handle the volume of calls from each conversation.
```
Comment (cshape) [inline faas-samples/current-queue-health/README.md:11]: I wonder if we have any guidance to offer on managing these trade-offs. 

Like, if they have minimal volume (less than 100 chats per day or something), would all of this be overkill and they can just query the queue health API directly? Or do we advise that, for any client, if they want to use the queue health or shift status in a bot, they should cache it like this?

File: faas-samples/shift-status/README.md [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+    updateConversationContextService(shiftStatusData, mavenApiKey);
+  }
+
+  main()
+}
+```
+
+This function does several things, but ultimately it logs a bot agent in, uses that bearer token to call the shift status API, passes that information to a function to format it, and then passes that data to a function to update the Context Session Store. Looking at this data in the Context Session Store using Postman, we see the following:
+
+<!--  POSTMAN RESPONSE IMAGE HERE -->
```
Comment (cshape) [inline faas-samples/shift-status/README.md:173]: Is there supposed to be an image here?

**Review by cshape — COMMENTED** [review]
Looks basically good but I had a couple questions before merging. @dahby

File: faas-samples/shift-status/README.md (hunk 2 above)
Comment (dahby) [reply faas-samples/shift-status/README.md:173]: Yes, I'll get it in there shortly

File: faas-samples/current-queue-health/README.md (hunk 1 above)
Comment (dahby) [reply faas-samples/current-queue-health/README.md:11]: All guidance I've received on this matter is that this is the proper way to get this information into the bots. While there are definitely brands out there accessing the information from the bot directly, we've been advised to promote this process as a best practice.

**Review by cshape — APPROVED** [review]
Looks good to me - merging.

**Pull request merged**